WeatherMonitor/Utilities/ConsoleUtilities.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Text;
+using WeatherMonitor.PrintConfig;
+namespace WeatherMonitor.Utilities;
+
+public abstract class ConsoleUtilities
+{
+    public static void StartApplication()
+    {
```

Comment:
It would be better to take responsibility for 'starting the application' somewhere else.
This Console Utility should only care for printing to and reading from the console.
Maybe have a wrapper logic in the `Program` or in a 'Runner' class or smth.